src/screens/SUE/SueReportScreen.tsx

```diff
@@ -199,11 +213,57 @@ export const SueReportScreen = ({ navigation }: { navigation: any }) => {
   };
   /* eslint-enable complexity */
 
-  const handleNextPage = () => {
+  useEffect(() => {
+    readReportValuesFromStore();
+  }, []);
+
+  const storeReportValues = async () => {
+    await addToStore('sueReportValues', { selectedPosition, serviceCode, ...getValues() });
+  };
+
+  const readReportValuesFromStore = async () => {
+    const storedValues = await readFromStore('sueReportValues');
+
+    if (storedValues) {
+      setStoredValues(storedValues);
+      setServiceCode(storedValues.serviceCode);
+      setSelectedPosition(storedValues.selectedPosition);
+      setValue('city', storedValues.city);
+      setValue('description', storedValues.description);
+      setValue('email', storedValues.email);
+      setValue('firstName', storedValues.firstName);
+      setValue('houseNumber', storedValues.houseNumber);
+      setValue('images', storedValues.images);
+      setValue('lastName', storedValues.lastName);
+      setValue('phone', storedValues.phone);
+      setValue('street', storedValues.street);
+      setValue('termsOfService', storedValues.termsOfService);
+      setValue('title', storedValues.title);
+      setValue('zipCode', storedValues.zipCode);
```

Comment:
wouldn't it be possible to go through all stored values and store them dynamically instead of defining each manually? the key is always the same for `setValue` as in `storedValues`